app/models/primer.rb

```diff
@@ -1,9 +1,9 @@
 class Primer < ActiveRecord::Base
 
   has_many :marker_assays_A, class_name: 'MarkerAssay',
```

Comment:
Is that capital A/B important? Since it looks so un-rdbms-y ;). But if that is required...